packages/app-extensions/src/formField/formField.js

```diff
@@ -44,24 +56,37 @@ export const formFieldFactory = (mapping, data, resources = {}) => {
     )
 
     const mandatory = !readOnly && _get(modelField, `validation.mandatory`, false)
+    const hasValue = value !== null && value !== undefined && value.length !== 0
```

Comment:
`value.length`: what types of values are handled here? Do they all provide a `length` property?

Reply:
It is not guaranteed that all values have a length property. Line 59 is a generic fallback and gets overwritten on line 18 by prop typeEditable.hasValue if provided. formField/typeEditable contain specific implementation to check if a value exist. Please contact @MethodenMann for further information.

Reply:
In this case, the `length` property should only be taken into account, if it's not undefined. For all values without `length` property, it should be sufficient if they aren't null or undefined.